01UML/src/Konto.java

```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+
+public class Konto {
+
+    private String bezeichnung;
+    private List<Kunde> zeichnungsberechtigter;
```

Comment:
Muss hier sicher gestellt werden dass immer mindestens ein Zahlungsberechtigter in der Liste ist? Diese Einschränkung könnte im Konstruktor implementiert werden.